src/api/app/controllers/statistics_controller.rb

```diff
@@ -48,16 +46,16 @@ def rating
 
       # try to get previous rating of this user for this object
       previous_rating = Rating.where('object_type=? AND object_id=? AND user_id=?', object.class.name, object.id, User.current.id).first
-      data = ActiveXML::Node.new(request.raw_post)
+      data = Xmlhash.parse(request.raw_post)
       if previous_rating
         # update previous rating
-        previous_rating.score = data.to_s.to_i
```

Comment:
I was tempted to remove this all together - it's not documented and not tested. And a XML document that is passed as .to_s.to_i made me wonder.

Reply:
https://trello.com/c/cED4FUV8